src/nginxconfig/generators/conf/nginx.conf.js

```diff
@@ -47,6 +47,12 @@ export default (domains, global) => {
         worker_connections: 65535,
     };
 
+    // Modules!
+    config.http.push(['# Load modules', '']);
+    config.http.push(['include', [
+        global.tools.modularizedStructure.computed ? `${global.nginx.nginxConfigDirectory.computed.replace(/\/+$/, '')}/modules-enabled/*` : '',
+    ].filter(x => x.length)]);
+
```

Comment:
I'm not really familiar with this type of template files, but I hope I copied the similar code for enabled sites correctly.